services/account_last_login_test.go

```diff
@@ -0,0 +1,27 @@
+package services_test
+
+import (
+	"testing"
+
+	"github.com/keratin/authn-server/services"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/keratin/authn-server/data/mock"
+)
+
+func TestLastLoginUpdater(t *testing.T) {
+	t.Run("last login", func(t *testing.T) {
```

Comment:
would you add a test for the ErrNotFound behavior? i don't mind if it's part of this `t.Run` or in another.